tests/testsuite/lints_table.rs

```diff
@@ -856,3 +856,51 @@ warning: `im_a_teapot` is specified
         )
         .run();
 }
+
+#[cargo_test]
+fn missing_field_level() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "foo"
+                version = "0.1.0"
+                edition = "2015"
+
+                [lints.rust]
+                unsafe_code = { }
+            "#,
+        )
+        .file("src/main.rs", "fn main() {}")
+        .build();
+
+    p.cargo("check")
+        .with_status(101)
+        .with_stderr_contains("[..]missing field `level`[..]")
```

Comment:
At least for myself, I prefer to not use the `contains` methods because they are brittle.  A slight change and you can be testing for the wrong condition.
(this is especially true for `does_not_contain`)

Reply:
Switch to `with_stderr`.